src/main/java/com/semmle/jira/addon/util/JiraUtils.java

```diff
@@ -196,7 +196,7 @@ public static void createConfigKeyCustomField() throws GenericEntityException {
       builder.setConfigurationItemAccessLevel(ConfigurationItemAccessLevel.LOCKED);
       builder.setSource(owner);
 
-      managedConfigurationItemService.updateManagedConfigurationItem(builder.build());
+      //      managedConfigurationItemService.updateManagedConfigurationItem(builder.build());
```

Comment:
commented out code?

Reply:
I guess left-over code from trying to test what happens if you remove the field.

Reply:
Whoops. Yep, spot on!

Fixed now...